packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -75,9 +77,9 @@ class CpuProfileData {
     final stackTraceEvents =
         (json[traceEventsKey] ?? []).cast<Map<String, dynamic>>();
     final samples = stackTraceEvents
-        .map((trace) => CpuSample.parse(trace))
+        .map((trace) => ProfileDataCpuSample.parse(trace))
```

Comment:
a couple other things you can do if you don't want to rename just to avoid a class name conflict
1) add a prefix for the import: `import 'package:vm_service/vm_service.dart' as vm_service;`. Then everywhere you need to use the CpuSample class from package:vm_service, you would write `vm_service.CpuSample`
2) hide the conflicting class when you import the library (this won't work in this case because you actually need to use the class with the conflicting name, but just adding it for your future information). `import 'package:vm_service/vm_service.dart' hide CpuSample;`. Then the classes would not conflict. You can also use the keyword `show` if you want to import only a single or a few select classes from a library.

Reply:
I can definitely put it back the way it was. This was more for readability as it was really confusing to be jumping around the code and then realize that I was looking at 2 different classes.